resources/views/contact.blade.php

```diff
@@ -3,28 +3,47 @@
 @section('title-block')Обратная связь@endsection
 
 @section('content')
-<h1>Обратная связь</h1>
-    <form action="{{ route('contact') }}" method="post">
-        @csrf
-
-        <div class="form-group">
-            <label for="name">Введите имя </label>
-            <input type="text" name="name" placeholder="Введите имя" id="name" class="form-control">
-        </div>
-        <br>
-        <div class="form-group">
-            <label for="email">Введите email </label>
-            <input type="text" name="email" placeholder="Введите email" id="email" class="form-control">
-        </div>
-        <br>
-        <div class="form-group">
-            <label for="phone">Введите телефон</label>
-            <input type="text" name="phone" placeholder="Введите телефон" id="phone" class="form-control">
-        </div>
-        <br>
-        <button type="submit" class="btn btn-success">Отправить</button>
-
-
-    </form>
+    <h1>Обратная связь</h1>
+
+
+    @if ($errors->any())
+        <div class="alert alert-danger">Данные внесены не верно!</div>
+    @endif
+
+
+    <div class="row">
+        <form action="{{ route('contact.store') }}" method="post">
+            @csrf
+
+            <div class="form-group">
+                <label for="name">{{ __('validation.attributes.name') }}</label>
+                <input value="{{ old ('name') }}" type="text" name="name" placeholder="Введите имя"
```

Comment:
можно вот так сделать
```suggestion
                <input value="{{ old('name') }}" type="text" name="name" placeholder="__('validation.attributes.name')"
```
ниже точно также можно сделать)

плюс форматирование, после `old` не должно быть пробела
ниже тоже самое